six/modules/c++/six.sicd/tests/test_mesh_roundtrip.cpp

```diff
@@ -0,0 +1,333 @@
+/* =========================================================================
```

Comment:
Is this something that's supposed to get pulled into the automated tests?

Reply:
Yes I think that's a good idea @JonathanMeans .  Can you make a PR with this added in?  FYI, I need to add something to `six_sicd.i` to get Python bindings to compile... working it.